mne/io/base.py

```diff
@@ -1997,6 +2004,16 @@ def _write_raw(fname, raw, info, picks, fmt, data_type, reset_range, start,
 
         pos_prev = pos
 
+    if raw.annotations is not None:
+        start_block(fid, FIFF.FIFFB_MNE_ANNOTATIONS)
+        write_int(fid, FIFF.FIFF_MNE_BASELINE_MIN, raw.annotations.onset)
+        write_int(fid, FIFF.FIFF_MNE_BASELINE_MAX, raw.annotations.duration)
```

Comment:
Okay, so could we use ``FIFF_MNE_ANNOTATION_DURATION = FIFF_MNE_BASELINE_MIN`` in ``constants``. Is there a risk that this overlap will cause trouble? Isn't annotations purely an attribute of raw and baseline for epochs and such?
Other option would be just introducing a couple of new constants. There are still plenty available.